google-ads-examples/pom.xml

```diff
@@ -21,15 +21,15 @@
   <parent>
     <groupId>com.google.api-ads</groupId>
     <artifactId>google-ads-parent</artifactId>
-    <version>1.1.1-SNAPSHOT</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
You don't need to change the versions in the POMs as well - the release script will handle that for you.

Reply:
My mistake, I have reverted the versions back to their original state.